config/analytics.yml

```diff
@@ -0,0 +1,3 @@
+shared:
+  candidates:
+    - created_at
```

Comment:
Just trying to remember how this achieves our mvp ... we wanted to know when a user signs-up, right? So by including this field we ensure we know when a candidate record is created? Is this the best way to do this?

This means we'll always send `created_at` ... but this is only useful for the sign-up event so sending it for every entity update afterwards wouldn't be useful; this field won't ever change. We could probably achieve this simply by emitting an entity-create event, which we would do by sending some other field on `candidate`, one that is more likely to change at some point.

Sorry if the above ramble is a bit naive.

Reply:
@misaka I think we should follow the same principle here as we did with including all of the data for each entity update event: we should assume that the initial data might have gone missing. So at any point we might want to know the *current* version of created_at, even though obviously it shouldn't ever change. Please can we include created_at in all entity events?

(I appreciate this is the complete opposite of how you'd want to handle this in the service itself! Loads of repetition and denormalisation seems to be the name of the game in analytics land.)

Reply:
No problem at all, I appreciate the discussion and feedback! What you've said makes sense, let's keep it in there!

Reply:
Thanks @stevenleggdfe. We currently only send entity events if there any analytics fields for the model — I assumed `created_at` was probably not controversial! Was not intended to carry the information about when they signed up — I hoped that the `occurred_at`, the table name, the `entity_created` type (and the `user_sign_up` tag for good measure) would do that.